packages/wonder-blocks-labeled-field/package.json

```diff
@@ -13,9 +13,8 @@
   "author": "",
   "license": "MIT",
   "publishConfig": {
-    "access": "restricted"
+    "access": "public"
```

Comment:
Referred to the button's package.json config:

https://github.com/Khan/wonder-blocks/blob/00a93181b1f1ec5e8be3793c8e9cf98cb3a7b015/packages/wonder-blocks-button/package.json#L5-L7